### ISSUE TITLE
[Jack Transport] Seq64 doesn't reposition jack transport

### ISSUE THREAD

**Issue by jean-emmanuel** [ahlstromcj/sequencer64#88]
Jack transport repositioning function is only called when seq64 is set as transport master, this doesn't conform to common usage : clients can relocate, the transport master's only privilegied controls are related to bpm and cycles lengths. Hydrogen's handling of jack transport is a good example we could follow, what do you think ?

**Comment by ahlstromcj** [comment]
Not sure. Hydrogen doesn't send out BPM until transport is stopped, from my
experiments.  Can you check out the latest wip, I think I may have broken
jack_assistant ....

**Comment by jean-emmanuel** [comment]
I'm using 0.9.7, this one does send bpm while playing (best way to check is with qjackctl). Do you have any thoughts on the first question (slave being able to relocate) ?
I'll check the wip branch today !

**Comment by ahlstromcj** [comment]
Just updated wip, got seq64 to stop resetting the BPM when you change it as JACK master.  Seems to work.  See the git log.  I still thinking that "JACK slaves" should not change tempo; see the links noted in the top banner comments in the jack_ass module (jack_assistant.cpp).  Needs some testing, and some more research into JACK transports conventions, if there are any.

**Comment by jean-emmanuel** [comment]
> I still thinking that "JACK slaves" should not change tempo;

I agree, it's the transport's position that's supposed to be changeable by slaves.

**Comment by jean-emmanuel** [comment]
From http://www.jackaudio.org/files/docs/html/transport-design.html

> Transport Control
> The JACK engine itself manages stopping and starting of the transport. Any client can make transport control requests at any time.

>  Repositioning
> These request a new transport position. They can be called at any time by any client. Even the timebase master must use them
>

**Comment by jean-emmanuel** [comment]
Also, when testing seq64's jack related things, note that there is a bug that currently prevents any jack-related change to actually work when done from the gui, you have to relaunch the app...

**Comment by ahlstromcj** [comment]
Depends.  You might be able to get away with going to the JACK tab (Ctrl-P?) and disconnect and reconnect.   I can't remember if that works for both midi and transport, or just for transport.
